src/xenia/gpu/command_processor.cc

```diff
@@ -734,6 +734,13 @@ bool CommandProcessor::ExecutePacketType3(RingBuffer* reader, uint32_t packet) {
       result = true;
       break;
     }
+    case PM4_WAIT_FOR_IDLE: {
```

Comment:
Would be useful to have a comment about its appearance in Duke Nukem Forever and steps to trigger it in the game.